error.go

```diff
@@ -121,9 +123,22 @@ func (e ErrWorkflow) Unwrap() []error {
 	}
 	return rv
 }
-func (e ErrWorkflow) IsNil() bool {
+func (e ErrWorkflow) AllSucceeded() bool {
+	for _, sErr := range e {
+		switch {
+		case sErr.Status == Succeeded && sErr.Err == nil:
```

Comment:
Any case status == succeeded but err != nil?

Reply:
There shouldn't be a case that Status == Succeeded but err != nil if everything goes well, though just a trivial check would be nice to have in my opinion